nameparser/config/titles.py

```diff
@@ -289,7 +290,13 @@
     'foreign',
     'forester',
     'founder',
+    'fr',
+    'frau',
```

Comment:
I would not be able to think of "Frau" being used as a first name. "Frauke" is a first name, but "Frau" used to be a sign for nobility (as in "Freifrau") in the middle ages, and trickled down to a common denominator of sex/gender lateron. So it would be common to be used in appellative function or when addressing a female person in writing.

Reply:
I these are all very legitimate and common titles.
Maybe a bit philosophical at this stage, but ... People can be named the strangest things, so requiring the categories to be distinct is unrealistic in my opinion. I think the role of this library should be to make a "best guess", probably not  ONLY based on whether it matches these data points. The amount words and categories of there are also very important factors that the library could take into account (in some future version). For example, I definitely know of people named `Prince` or `Sheikh` as their  first names. So if we come across such words, we probably need to look at the context to make a best guess about what it means. Same thing applies to the prefixes like "Van" which are very common prefixes but also first names and possibly (unlikely IMO) middle names.